CRM/Stripe/Upgrader.php

```diff
@@ -51,4 +51,23 @@ public function upgrade_1_9_003() {
     }
     return TRUE;
   }
+
+  /**
+   * Add processor_id column to civicrm_stripe_customers table.
+   *
+   * @return TRUE on success
+   * @throws Exception
+   */
+  public function upgrade_4_6_01() {
```

Comment:
Then again, maybe not - since CiviCRM just tracks which upgrade(s) have been performed, and we don't want to try and run it twice.